packages/firestore/src/util/async_queue.ts

```diff
@@ -143,26 +210,14 @@ export class AsyncQueue {
    * scheduled with a delay can be rejected or queued for immediate execution.
    */
   drain(executeDelayedTasks: boolean): Promise<void> {
```

Comment:
As a future cleanup, it seems like we could now get rid of this method. Each caller could invalidate its tasks when we call .shutdown().

Reply:
Not sure how that works with the executeDelayedTasks codepath...  which I think we'll actually want to extend to be able to test timeouts better...